server/roomlogs.ts

```diff
@@ -222,42 +239,73 @@ export class Roomlog {
 		this.broadcastBuffer.push(fullMessage);
 	}
 	parseChatLine(line: string) {
-		const messageStart = !this.noLogTimes ? '|c:|' : '|c|';
-		const section = !this.noLogTimes ? 4 : 3; // ['', 'c' timestamp?, author, message]
-		if (line.startsWith(messageStart)) {
-			const parts = Utils.splitFirst(line, '|', section);
-			return {user: parts[section - 1], message: parts[section]};
+		const prefixes = [['|c:|', 4], ['|c|', 3]] as [string, number][];
+		for (const [messageStart, section] of prefixes) {
+			// const messageStart = !this.noLogTimes ? '|c:|' : '|c|';
+			// const section = !this.noLogTimes ? 4 : 3; // ['', 'c' timestamp?, author, message]
+			if (line.startsWith(messageStart)) {
+				const parts = Utils.splitFirst(line, '|', section);
+				return {user: parts[section - 1], message: parts[section]};
+			}
 		}
 	}
 	roomlog(message: string, date = new Date()) {
-		if (!this.roomlogStream) return;
-		const timestamp = Chat.toTimestamp(date).split(' ')[1] + ' ';
 		message = message.replace(/<img[^>]* src="data:image\/png;base64,[^">]+"[^>]*>/g, '');
```

Comment:
```suggestion
		message = message.replace(/<img[^>]* src="data:image\/png;base64,[^">]+"[^>]*>/g, '[img]');
```

unrelated but maybe we shouldn't strip images out without a note

Reply:
Okay probably this should be a different commit/PR